sdk/nodejs/tests/runtime/langhost/cases/062.providerref_invokes/index.js

```diff
@@ -0,0 +1,37 @@
+// Test the ability to invoke provider functions via RPC.
+
+let assert = require("assert");
+let pulumi = require("../../../../../");
+
+(async () => {
```

Comment:
pattern to show how users could get a provider-ref asynchronously to pass along.